project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.12
```

Comment:
We should bump this to 1.2.8, but that should probably be a follow-up PR.

Reply:
Ok, bumped it.